crates/rattler_conda_types/src/snapshots/rattler_conda_types__prefix_record__test__pysocks-1_7_1-pyh0701188_6_json.snap

```diff
@@ -14,6 +13,7 @@ license_family: BSD
 md5: 56cd9fe388baac0e90c7149cfac95b60
 name: pysocks
 noarch: python
+optional_depends: {}
```

Comment:
mm strange, since you made it so that empty optional dependencies are skipped I would not expect this here.